tests/test_hardhat.py

```diff
@@ -94,19 +94,19 @@ def test_set_block_gas_limit(hardhat_provider):
     assert hardhat_provider.set_block_gas_limit(gas_limit) is True
```

Comment:
Everywhere we are calling `hardhat_provider._web3.eth.get_block("latest")` in this file can be replaced with `hardhat_provider.get_block("latest")`

Reply:
Will do